### PR TITLE
Add sinch component

### PR DESCRIPTION
**Pull request by bendikrb**
**Description:**
Documentation for the Sinch SMS component.

**Pull request in home-assistant:** home-assistant/home-assistant#26502

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sinch.markdown [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# Using default recipients
+notify:
+  - platform: sinch
+    name: Sinch
+    service_plan_id: SINCH_SERVICE_PLAN_ID
+    api_key: SINCH_API_KEY
+    default_recipient: [PHONE_NO1, PHONE_NO2]
+```
+
+```
```
Comment (frenck) [inline source/_components/sinch.markdown]: This is not a code block...

File: source/_components/sinch.markdown [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Configuration
+
+To add Sinch to your installation, add the following to your Home Assistant `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - platform: sinch
+    name: Sinch
```
Comment (frenck) [inline source/_components/sinch.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/sinch.markdown [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Sinch SMS"
+description: "Instructions on how to add Sinch notifications to Home Assistant."
+logo: sinch.png
+ha_category:
+  - Notifications
+ha_release: 0.99.1
+redirect_from:
```
Comment (frenck) [inline source/_components/sinch.markdown]: This is a new page, no redirection is needed. Please remove

File: source/_components/sinch.markdown [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Sinch SMS"
+description: "Instructions on how to add Sinch notifications to Home Assistant."
+logo: sinch.png
+ha_category:
+  - Notifications
+ha_release: 0.99.1
```
Comment (frenck) [inline source/_components/sinch.markdown]: New components are never added in a minor release.
```suggestion
ha_release: "0.100"
```

File: source/_components/sinch.markdown [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - platform: sinch
+    name: Sinch
+    service_plan_id: SINCH_SERVICE_PLAN_ID
+    api_key: SINCH_API_KEY
+
+# Using default recipients
```
Comment (frenck) [inline source/_components/sinch.markdown]: Additional examples must go in their own section.

**Review by frenck — DISMISSED** [review]
Hi there, @bendikrb! I've reviewed your PR and left some comments. Could you please take a look? Thanks! 👍

File: source/_components/sinch.markdown (hunk 1 above)
Comment (bendikrb) [reply source/_components/sinch.markdown]: Dang, added just for readability in my editor. Fixed!

File: source/_components/sinch.markdown [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+### Full configuration example
+```yaml
+# Example configuration.yaml entry
+notify:
+  - platform: sinch
+    name: Sinch
+    service_plan_id: SINCH_SERVICE_PLAN_ID
+    api_key: SINCH_API_KEY
+    default_recipient: [PHONE_NO1, PHONE_NO2]
+    sender: Home assistant
```
Comment (klaasnicolaas) [inline source/_components/sinch.markdown]: `sender` isn't required

File: source/_components/sinch.markdown [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+### Full configuration example
+```yaml
+# Example configuration.yaml entry
+notify:
+  - platform: sinch
+    name: Sinch
+    service_plan_id: SINCH_SERVICE_PLAN_ID
+    api_key: SINCH_API_KEY
+    default_recipient: [PHONE_NO1, PHONE_NO2]
```
Comment (klaasnicolaas) [inline source/_components/sinch.markdown]: `default_recipient` isn't required

File: source/_components/sinch.markdown [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+    service_plan_id: SINCH_SERVICE_PLAN_ID
+    api_key: SINCH_API_KEY
+```
+
+### Full configuration example
+```yaml
+# Example configuration.yaml entry
+notify:
+  - platform: sinch
+    name: Sinch
```
Comment (klaasnicolaas) [inline source/_components/sinch.markdown]: `name` isn't required

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**